lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -246,6 +319,14 @@ namespace Js
     THROW_ERROR_IMPL(ThrowWebAssemblyCompileError, CreateWebAssemblyCompileError, GetWebAssemblyCompileErrorType, kjstWebAssemblyCompileError)
     THROW_ERROR_IMPL(ThrowWebAssemblyRuntimeError, CreateWebAssemblyRuntimeError, GetWebAssemblyRuntimeErrorType, kjstWebAssemblyRuntimeError)
     THROW_ERROR_IMPL(ThrowWebAssemblyLinkError, CreateWebAssemblyLinkError, GetWebAssemblyLinkErrorType, kjstWebAssemblyLinkError)
+
+    void __declspec(noreturn) JavascriptError::ThrowAggregateError(ScriptContext* scriptContext, JavascriptArray* errors)
+    {
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        JavascriptError* pError = library->CreateAggregateError();
+        JavascriptError::SetErrorsList(pError, errors, scriptContext);
+        JavascriptExceptionOperators::Throw(pError, scriptContext);
```

Comment:
Should either update this to include an hCode + a call to SetErrorMessage OR delete this function and just do the steps inline when needed.